docker/fullstack/entry.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
+export FRONTEND_URL="${FRONTEND_URL:-${HYPERDX_APP_URL:-http://localhost}:${HYPERDX_APP_PORT:-8080}}"
+
+echo "Visit the HyperDX UI at $FRONTEND_URL"
+echo ""
+
 # Use concurrently to run both the API and App servers
 npx concurrently \
   "--kill-others" \
```

Comment:
just ux feedback for later, ctrl+c doesn't seem to kill this script, but it works fine in old local mode. I'm wondering if some of the `wait -n` stuff can help? https://github.com/hyperdxio/hyperdx/blob/main/docker/local/entry.sh#L85 (I forgot how it was fixed there)

Reply:
good catch. pushed the fix